netsocadmin/config.py

```diff
@@ -18,6 +18,8 @@
     "zsh": "/usr/bin/zsh"
 }
 
+VALID_USERNAME = r"^[a-z0-9]([a-z0-9\-\_]{0,60}[a-z0-9])$"
```

Comment:
Should have a `?` after the parentheses to have 0 or 1 or the group

Reply:
usernames are limited to at least 2 characters, cant start or end with underscore or hyphen

Reply:
Okay in that case you don't need the parentheses at all

Reply:
Also if you want capitals available you need to add `A-Z` in each of the square brackets